<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomScreen</class>
 <widget class="QMainWindow" name="RoomScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="insideRoomLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="files.qrc">:/image/images/backgrounds/inside.jpg</pixmap>
    </property>
   </widget>
   <widget class="QTextBrowser" name="osanaText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>740</width>
      <height>105</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser {
	font: 75 16pt &quot;メイリオ&quot;;
}</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'メイリオ'; font-size:16pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;おはよう&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="osanaLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>371</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="files.qrc">:/image/images/character/figure1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="serifButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>140</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background: transparent;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="finishButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>410</y>
      <width>140</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background: transparent;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="windowLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>405</y>
      <width>760</width>
      <height>182</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="files.qrc">:/image/images/windows/textbox_serif.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="timerButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>410</y>
      <width>140</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background: transparent;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="breakButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>140</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background: transparent;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="finishLabel">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>400</y>
      <width>140</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="files.qrc">:/image/images/windows/button_finish.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>insideRoomLabel</zorder>
   <zorder>osanaLabel</zorder>
   <zorder>finishLabel</zorder>
   <zorder>windowLabel</zorder>
   <zorder>osanaText</zorder>
   <zorder>timerButton</zorder>
   <zorder>serifButton</zorder>
   <zorder>finishButton</zorder>
   <zorder>breakButton</zorder>
  </widget>
 </widget>
 <resources>
  <include location="files.qrc"/>
 </resources>
 <connections/>
</ui>
